### DISCUSSION TITLE
Can SVG images be used for the application logo?

### DISCUSSION THREAD

**Discussion by Juriyx** [unoplatform/uno#14455] [Q&A]
Hello. I found [this article about SVG usage in Uno projects](https://platform.uno/docs/articles/features/svg.html).

Do I understand right that SVG images can be used for the application logo? And how is this achieved?

**Comment by jeromelaban** [comment]
Thanks for the question! 

Have you taken a look at https://platform.uno/docs/articles/external/uno.resizetizer/doc/using-uno-resizetizer.html?

**Reply by xperiandri** [reply]
Confirm, if I understand it correctly. No platform can directly draw SVG as an icon. For that reason, Uno creates the right PNGs for each platform
